Palo Alto Networks/paloalto-cortex/ingest/parser.yml

```diff
@@ -0,0 +1,37 @@
+name: cortex-xdr
+pipeline:
+  - name: parsed_event
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: setter
+    description: ''
+    filter: ''
+    external: null
+stages:
+  setter:
+    actions:
+      - set:
+          cortexxdr.shost: '{{parsed_event.message.shost}}'
+          cortexxdr.suser: '{{parsed_event.message.suser}}'
+          cortexxdr.action: '{{parsed_event.message.act|lower}}'
+          cortexxdr.category: '{{parsed_event.message.cat|upper}}'
+          cortexxdr.event_subtype: '{{parsed_event.message.subtype}}'
+          cortexxdr.severity_human: '{{parsed_event.message.Severity}}'
+          cortexxdr.original_message: '{{parsed_event.message.msg}}'
```

Comment:
Original message is already present in the resulting event (in the message field).
```suggestion
```